api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1338,6 +1338,8 @@ type AWSPlatformSpec struct {
 
 	// MultiArch specifies whether the Hosted Cluster will be expected to support NodePools with different
 	// CPU architectures, i.e., supporting arm64 NodePools and supporting amd64 NodePools on the same Hosted Cluster.
+	// Deprecated: This field is no longer used. The HyperShift Operator now performs multi-arch validations despite the
+	// platform type.
```

Comment:
What is the upgrade story for users already relying on this field?

Reply:
This flag was previously used to validate a multi-arch image was used for the HC when this flag was enabled. We perform multi-arch validation between the HC release image and the NodePool CPU in the NodePool controller now.

Reply:
They don't have to set any value. It won't matter what value they set for this since the flag is not used in the code at all.

Reply:
When the user has configured here to use a multi-arch payload, how do they represent that in the new API?

When the user has configured here _not_ to use a multi-arch payload, how do they represent that in the new API?